tests/pytests/test.py

```diff
@@ -2588,7 +2588,7 @@ def testMod_309(env):
     for i in range(n):
         conn.execute_command('HSET', f'doc{i}', 'test', 'foo')
     waitForIndex(conn, 'idx')
-    res = env.cmd('FT.AGGREGATE', 'idx', 'foo')
+    res = env.cmd('FT.AGGREGATE', 'idx', 'foo', 'TIMEOUT', 0)
```

Comment:
Consider a large value which will not be flaky but still not zero, so will not need to wait till test fails due to exceeded 30m without output.

Reply:
This should be its own PR anyway. removing.